mdbook/src/05-meet-your-software/debug-it.md

```diff
@@ -33,7 +33,7 @@ The `../../..` in this command is needed, since each example project is in a "wo
 contains the entire book. Workspaces have a single shared `target` directory. Check out [Workspaces
 chapter in Rust Book] for more.
 
-> **NOTE** If `cargo-embed` prints a lot of warnings here don't worry about it. As of now it does
+> **NOTE**: If `cargo-embed` prints a lot of warnings here don't worry about it. As of now it does
```

Comment:
The convention in this book is to leave the colon `:` off. I agree that it's a bit janky, but it's everywhere.

Reply:
I pattern matched from this file and didn't take a look at the rest, but this file mostly has the with-colon version! Will change, though.